bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -90,23 +90,29 @@ def construct_wtperf_command_line(wtperf: str, env: str, test: str, home: str, a
         command_line.append(home)
     return command_line
 
+def to_value_list(reported_stats: List[PerfStat], brief: bool):
```

Comment:
I was wondering why you moved `to_value_list` from `PerfStatCollection`?

Reply:
I've scoped down `PerfStatCollection` to only apply to a single test run, so now for a batch_file containing 3 different tests a `PerfStatCollection` is created per test. 
Rather than try to aggregate the results of all runs into a single `PerfStatCollection` object the relevant info is stored in a List[PerfStat] which is then passed into `detailed_perf_stats`/`brief_perf_stats`. 
Given that `to_value_list` is only called in these functions which no longer require a `PerfStatCollection` I've pulled it out into `wtperf_main.py`